.env.development

```diff
@@ -38,9 +38,10 @@ VITE_MODE_LIB=false
 VITE_MYMAPS_URL="https://migration.geoportail.lu/mymaps"
 VITE_ARROW_MODEL_URL="https://migration.geoportail.lu/static-ngeo/models/arrow5.glb"
 VITE_ELEVATION_URL="https://migration.geoportail.lu/raster"
+VITE_PROFILE_URL="https://migration.geoportail.lu/profile.json"
 
 # Auth
-VITE_CREDENTIALS_ORIGIN="include"
+VITE_CREDENTIALS_ORIGIN="same-origin"
```

Comment:
Could you refresh my memory, why you changed this value?

Reply:
`"same-origin"` is safe and should be the default value everywhere. Of course this has no impact if value is "include" in `.env.development` but i would rather keep same-origin value in the main branch.

You can find the doc in the README.md file: https://github.com/Geoportail-Luxembourg/luxembourg-geoportail/blob/main/README.md#-authenticate-user